### DISCUSSION TITLE
app build for pwa in shared hosing returning 404 while hard reloading ther page

### DISCUSSION THREAD

**Discussion by vivek-Biplop** [quasarframework/quasar#16259] [CLI - PWA mode]
recently i deployed my quasar app to shared hosting and the problem is when i just reload the page with ctl+R it works but when i do hard refresh the page with ctl+shift+r it displays 404

```

404
Not Found
The resource requested could not be found on this server!
```

here is the test website url
https://baluwa.rgbitsolution.com.np

what can be the solution for this issue 

app build using ```quasar build -m pwa```

**Comment by metalsadman** [comment]
https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations
